src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -599,4 +605,40 @@ private String getElementSrcWithRetryAfterWaitForPresence(By by) {
         }
     }
 
+    /**
+     * Expands a particular panel, causing its results to load.
```

Comment:
... particular **question** panel... ? Not sure if this is used in places where the panel doesn't represent a question though; do let me know.

Reply:
This hasn't been addressed yet.

Reply:
Its only used for question panel